### DISCUSSION TITLE
Which shell is used?

### DISCUSSION THREAD

**Discussion by fortierq** [community/community#25627] [Actions]
<p>Which shell are Github Action using?<br>
It looks like it is not bash, since if [[ … ]] does not work, for example.<br>
Is it sh, dash?<br>
Is it possible to use bash instead?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It should be using Bash by default <em>if</em> you’re not on a Windows runner and it’s available (e.g. some container images might not have it). You can always select a specific shell with the <code>shell</code> parameter for the <code>run</code> step.</p>
<p>You can see the defaults and more details here: <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell">Using a specific shell</a></p>
